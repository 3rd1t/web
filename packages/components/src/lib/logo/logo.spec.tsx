import React from "react"

import Logo from "./logo"
import { render } from "@testing-library/react"
describe("Logo", () => {
  describe("with domain", () => {
    it("renders correctly", () => {
      const { baseElement } = render(<Logo color="text-blue-500" domain="domain" />)
      expect(baseElement).toBeTruthy()
    })
  })
  describe("without domain", () => {
    it("renders correctly", () => {
      const { baseElement } = render(<Logo />)
      expect(baseElement).toBeTruthy()
    })
  })
})
