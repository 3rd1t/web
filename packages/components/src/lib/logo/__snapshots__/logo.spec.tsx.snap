// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` Logo with domain renders correctly 1`] = `
<div
  className="flex items-center space-x-2"
>
  <svg
    className="h-6 text-gray-900 stroke-current"
    fill="none"
    viewBox="0 0 194 148"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M9 106.208L97.3883 17.8199L134.158 54.5894M185.07 41.8615L96.6814 130.25L59.9118 93.4803"
      strokeWidth="25"
    />
  </svg>
  <div
    className="text-xl font-semibold"
  >
    <span
      className="text-gray-800"
    >
      PERFOLIO
    </span>
    <span
      className="uppercase text-blue-500"
    >
      domain
    </span>
  </div>
</div>
`;

exports[` Logo without domain renders correctly 1`] = `
<div
  className="flex items-center space-x-2"
>
  <svg
    className="h-6 text-gray-900 stroke-current"
    fill="none"
    viewBox="0 0 194 148"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M9 106.208L97.3883 17.8199L134.158 54.5894M185.07 41.8615L96.6814 130.25L59.9118 93.4803"
      strokeWidth="25"
    />
  </svg>
  <div
    className="text-xl font-semibold"
  >
    <span
      className="text-gray-800"
    >
      PERFOLIO
    </span>
  </div>
</div>
`;
